README.md

```diff
@@ -0,0 +1,103 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# The Budget
+>This is a mobile web application build with Ruby on Rails. Users can keep track of their expenses, and add them to their desired categories. 
+
+## Built With
+
+- Ruby
+- Ruby on Rails
+- Git
+- Postgresql
+
+---
+## Getting Started
+
+> To get a local copy up and running please follow these steps:
+
+- Open the command prompt in this directory.
+
+- Clone of the Repo:
+
+      $ git clone https://github.com/adamteddychang/The-Budget.git
+
+- Go to the repository folder in your command prompt and navigate into the project file:
+
+      $ cd The-Budget
+
+- Install the Ruby on Rails Gems:
+
+      $ bundle install or bundle i
+
+- To analyze and format the applictaion code run:
+
+      $ rubocop -A
+
+- Create databases
+
+      $ rails db:setup
+
+
+      $ rails db:migrate
+
+- Start the server:
+
+      $ rails s
+
+---
+
+
+
+## To run the tests
+
+- Install rspec:
+
+- and
+
+      $ rails generate rpesc:install
+
+- run all rspec tests:
+
+      $ rspec spec
+
+
+---
+### Prerequisites
+
+- Ruby
+- Ruby on Rails
+- Postgresql
+## Authors
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.